Gems/ROS2/Code/Source/VehicleDynamics/DriveModels/SkidSteeringDriveModel.h

```diff
@@ -38,5 +38,7 @@ namespace ROS2::VehicleDynamics
         SkidSteeringModelLimits m_limits;
         AZStd::unordered_map<AZ::EntityId, AZ::EntityComponentIdPair> m_wheelsData;
         VehicleConfiguration m_config;
+        float m_currentLinearVelocity = 0;
+        float m_currentAngularVelocity = 0;
```

Comment:
nit: Prefer `0.0f` for zero floating point literals if possible

Reply:
Adjusted, thanks.